pom.xml

```diff
@@ -147,12 +147,17 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
```

Comment:
hmm for some reason it selects ice4j 1.0 instead of ice4j 1.1 even if I bump the version in libjitsi. There is also jitsi-netaddr which depends on ice4j, but it does not have any version specified.